llvm/lib/Target/Mips/MipsISelLowering.cpp

```diff
@@ -4331,14 +4331,6 @@ MipsTargetLowering::isOffsetFoldingLegal(const GlobalAddressSDNode *GA) const {
   return false;
 }
 
-EVT MipsTargetLowering::getOptimalMemOpType(
```

Comment:
This hunk is a leftover from a previous version of the patch; please ignore when reviewing. I'll restore this.

(The implementation looks a bit suspect in how it ignores alignment completely and doesn't check the `strict-align` target feature. But that's a completely separate issue.)

Reply:
MIPS can use pairs of LWL/LWR LDL/LDR for unaligned accesses, so i64/i32 is always legal here even if unaligned accesses are not possible. (As long as the lowering takes this into account, which I believe it does).